server/gokbg3/grails-app/views/apptemplates/_tipp.gsp

```diff
@@ -62,6 +62,13 @@
                             config='KBComponent.EditStatus'/>
     </dd>
 
+    <dt>
+        <g:annotatedLabel owner="${d}" property="language">Language</g:annotatedLabel>
+    </dt>
+    <dd>
+        <g:xEditableRefData owner="${d}" field="language" config='KBComponent.Language'/>
```

Comment:
All other templates are using the category definition from KBComponent directly. Best to do it here to for consistencies' sake.

Reply:
I can see no inconsistency with `editStatus`. Please explain.

Reply:
I meant the other templates adding `language`, where you defined the `config` value as `${org.gokb.cred.KBComponent.RD_LANGUAGE}` rather than the resolved string used in this case.